db/Pos/DB/Class.hs

```diff
@@ -80,7 +80,7 @@ import           Control.Monad.Trans.Lift.Local (LiftLocal (..))
 import           Control.Monad.Trans.Resource   (ResourceT)
 import           Data.Conduit                   (Source)
 import qualified Database.RocksDB               as Rocks
-import qualified Ether
+import           Ether.Internal                 (HasLens (..))
```

Comment:
It turned out to be difficult to remove in a single task, this PR is concerned with `MonadReader` only.